src/cmd/commit.ts

```diff
@@ -26,7 +27,12 @@ const commitAndPush = async (values: Record<string, any>, branch: string): Promi
   const d = terminal(`cmd:${cmdName}:commitAndPush`)
   d.info('Committing values')
   const argv = getParsedArgs()
-  const message = isCi ? 'updated values [ci skip]' : argv.message || 'otomi commit'
+  let message: string
+  if (isCi && existsSync(`${env.ENV_DIR}/.rerun`)) {
+    message = '[apl-trigger]'
+  } else {
+    message = isCi ? 'updated values [ci skip]' : (argv.message as string) || 'otomi commit'
```

Comment:
This makes it we dont check double isCi:
```suggestion
    message = 'updated values [ci skip]'
```